deploy/terraform/digitalocean/variables.tf

```diff
@@ -0,0 +1,33 @@
+# DigitalOcean API Token. This can also be set via the
+# TF_VAR_do_token environment variable
+variable "do_token" {}
+
+variable "region" {
+  description = "The DigitalOcean region to deploy the k8s cluster into"
+  type        = string
+  default     = "nyc1"
+}
+
+variable "cluster_name" {
+  description = "Kubernetes cluster name"
+  type        = string
+  default     = "ecommerce"
+}
+
+variable "node_pool_name" {
+  description = "Name of the Kubernetes worker pool nodes"
+  type        = string
+  default     = "worker"
+}
+
+variable "node_size" {
+  description = "Cluster node size. See https://slugs.do-api.dev/ for slug options."
+  type        = string
+  default     = "s-2vcpu-2gb"
+}
+
+variable "node_count" {
+  description = "Number of nodes in the Kubernetes pool"
+  type        = number
+  default     = 3
```

Comment:
Isn't 3 nodes a bit too much for the e-commerce app as default?

Reply:
I can try smaller deployments. When I shrunk the droplet size down I was having some pod eviction issues. Will try tweaking this more to see if I can get a more sensible default.